catalog/tests/dags/data_refresh/test_remap_table_constraints.py

```diff
@@ -0,0 +1,79 @@
+from textwrap import dedent
+
+import pytest
+
+from data_refresh.remap_table_constraints import (
+    ConstraintInfo,
+    generate_constraints_for_table,
+)
+
+
+@pytest.mark.parametrize(
+    "all_constraints, table_name, temp_table_name, expected_alter_statements",
+    [
+        # No constraints
+        ([], "audio", "temp_import_audio", []),
+        (
+            [
```

Comment:
I know it's a little pedantic, but all of these constraints are bundled into the same test case. Would it be possible to separate these out into individual test cases, so they can each fail individually rather than as a single test if one should fail? We can use `audio` and `temp_import_audio` as the `table_name` and `temp_table_name` respectively for every test, so those at least wouldn't need to be parametrized.

Reply:
Part of what this is testing is that the generated constraints are ordered correctly when there are multiple, but I can add additional tests for individual constraints.

Reply:
Ahhh okay, thanks for that context!